modules/power_z_unsupported_known_issues.adoc

```diff
@@ -0,0 +1,18 @@
+[id="power z known issues_{context}"]
+== IBM Z and LinuxONE known issues. 
+
+  * Only supported on {product-title} 4.6 
```

Comment:
I would move this to a sub bullet in the ossm-supported-configurations file at line twenty.

* This release of {ProductName} is only available on {product-title} x86_64, IBM Z, and IBM Power Systems.
* * IBM Z is only supported on {product-title} 4.6 and later.
* * IBM Power Systems is (are?) only supported on {product-title} 4.6 and later.

Reply:
I would leave "is" here although I know it seems strange "IBM Power Systems" is the family name. 

IBM Power Systems is only supported on {product-title} 4.6 and later.

Reply:
I figured you would know what the standards are for the product names better than I do.  =)